Styles/components.css

```diff
@@ -487,23 +533,34 @@ body {
   margin: 0.2em;
 }
 .nav-item:hover {
-  background-color: #30363f;
+  background-color: #10f59d18;
+}
+.navbar-right {
+  justify-content: space-between;
+}
+.navbar-left {
+  justify-content: flex-start;
+}
+.navbar-center {
+  justify-content: center;
 }
 
 /* ************************Modal*************************************** */
 
 .modal {
   position: relative;
   padding: 2rem 1rem;
-  /* position: absolute; */
-  border: 1px ridge #4d6a7c;
+  border: 1px solid var(--myColor);
   border-radius: 0.35em;
-  box-shadow: 4px 6px 3px #292e36, -4px -3px 4px #313640;
+  border-radius: 8px;
+  background: linear-gradient(145deg, #ffffff, #e1e1e1);
```

Comment:
I just searched for this linear gradient with exact values in your code,too much repetitions define it in a variable and use it